README.md

```diff
@@ -1 +1,4 @@
-# Butterfly-Project
\ No newline at end of file
+# Butterfly-Project
+
+ Thanks to:
+ Chalk and Rootel ("More Faith Icons") for additional religion icons.
```

Comment:
Did you get permission for that? If yes, I don't plan to have a Credits section in the readme, it will be on the forums.

Reply:
Yes, I inquired on Sept 6 on the mod's steam page:

Chalk  [author] Sep 6 @ 5:01pm 
@LT. Rascek yes, you can use this if you attribute me (and Rootel) :)

I put there for reference so I wouldn't forget.

Reply:
Ok

Reply:
You can remove this now, I have added it to the second post: https://forum.paradoxplaza.com/forum/threads/wip-a-migrational-era-mod-for-ck3.1414709/#post-26850337